subprojects/snapshots/src/test/groovy/org/gradle/internal/snapshot/impl/DirectorySnapshotterTest.groovy

```diff
@@ -152,6 +168,14 @@ class DirectorySnapshotterTest extends Specification {
         !defaultExcludes.excludeFile('.svnsomething')
         !defaultExcludes.excludeFile('#some')
     }
+
+    private static String fileSystemRoot() {
```

Comment:
os independent root search

Reply:
Should I use `File.listRoots()` instead?

Reply:
How about `Path.getRoot()`?

Reply:
It's better, thanks.